client/components/pages/files/FilesTable.vue

```diff
@@ -260,6 +263,23 @@ function RenderIcon(props: { sorted: boolean; sortOrder: boolean }) {
         <PSkeleton v-else width="8rem" height="1rem" />
       </template>
     </PColumn>
+
+    <PColumn class="w-3em">
+      <template #body="slotProps">
+        <PButton
+          severity="secondary"
+          text
+          rounded
+          icon="i-tabler-dots-vertical"
+          size="small"
+          @click="(e) => {
+            e.stopPropagation()
+            openRowContextMenu(e)
+            selectedRow = slotProps.data
+          }"
```

Comment:
do make use of [event modifiers](https://vuejs.org/guide/essentials/event-handling.html#event-modifiers)

```suggestion
          @click.stop="(e) => {
            openRowContextMenu(e)
            selectedRow = slotProps.data
          }"
```